processing/src/main/java/org/apache/druid/query/scan/ScanQueryEngine.java

```diff
@@ -266,7 +261,7 @@ public void cleanup(Iterator<ScanResultValue> iterFromMake)
   private long calculateLimit(ScanQuery query, ResponseContext responseContext)
   {
     if (query.getOrder().equals(ScanQuery.Order.NONE)) {
-      return query.getLimit() - (long) responseContext.get(ResponseContext.CTX_COUNT);
+      return query.getLimit() - (long) responseContext.get(ResponseContext.Key.COUNT);
     }
     return query.getLimit();
```

Comment:
Please rename this property to "scanRowsLimit" for clarity.

Reply:
Renamed